admin_interface/templates/admin/base_site.html

```diff
@@ -10,22 +10,109 @@
+            --admin-interface-env-color: {{ theme.env_color }};
+            --admin-interface-header-background-color: {{ theme.css_header_background_color }};
+            --admin-interface-header-text-color: {{ theme.css_header_text_color }};
+            --admin-interface-header-link-color: {{ theme.css_header_link_color }};
+            --admin-interface-header-link_hover-color: {{ theme.css_header_link_hover_color }};
+            --admin-interface-module-background-color: {{ theme.css_module_background_color }};
+            --admin-interface-module-background-selected-color: {{ theme.css_module_background_selected_color }};
+            --admin-interface-module-text-color: {{ theme.css_module_text_color }};
+            --admin-interface-module-link-color: {{ theme.css_module_link_color }};
+            --admin-interface-module-link-selected-color: {{ theme.css_module_link_selected_color }};
+            --admin-interface-module-link-hover-color: {{ theme.css_module_link_hover_color }};
+            --admin-interface-generic-link-color: {{ theme.css_generic_link_color }};
+            --admin-interface-generic-link-hover-color: {{ theme.css_generic_link_hover_color }};
+            --admin-interface-save-button-background-color: {{ theme.css_save_button_background_color }};
+            --admin-interface-save-button-background-hover-color: {{ theme.css_save_button_background_hover_color }};
+            --admin-interface-save-button-text-color: {{ theme.css_save_button_text_color }};
+            --admin-interface-delete-button-background-color: {{ theme.css_delete_button_background_color }};
+            --admin-interface-delete-button-background-hover-color: {{ theme.css_delete_button_background_hover_color }};
+            --admin-interface-delete-button-text-color: {{ theme.css_delete_button_text_color }};
+            --admin-interface-related-modal-background-color: {{ theme.related_modal_background_color }};
+            --admin-interface-related-modal-background-opacity: {{ theme.related_modal_background_opacity }};
+
+        {% if theme.css_header_background_color != theme.css_module_background_color %}
+            --admin-interface-main-border-top: 10px solid var(--admin-interface-module-background-color);
```

Comment:
All variables defined with a condition should have a fallback value set in the `else` block of the condition.

Reply:
@fabiocaccamo 
yes its more clean to have a fallback value, it my be add in `else` bock here, or in the css file itself like

[source: w3schools Syntax of the var() Function](https://www.w3schools.com/css/css3_variables.asp)
`var(--name, value)` where: 
- `name` Required. The variable name (must start with two dashes)
- `value` Optional. The fallback value (used if the variable is not found)

kindly take a look at [mozilla: What happens with invalid variables?](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#what_happens_with_invalid_variables)

I've depended on that the default fallback behaviour will be:
1-  trying to inheritable from parent element
2-  use default initial value
and i didn't notice any breaks but at all it will be more better to have a clear defined fallback value if its needed